src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
-
 // write your code here
+const personsTable = people.map(person => {
+  return `
+    <tr>
+      <td>${person.name}</td>
+      <td>${person.sex === 'm' ? 'male' : 'female'}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.floor(person.died / 100)}</td>
+    </tr>`;
+});
+
+const table = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard tbody');
```
строки с людьми должны быть светлого цвета, а у тебя вся таблица приняла оранжевый стиль thead. 
Добавь селектор tbody и все должно быть ок.